src/XrdPlugins.cmake

```diff
@@ -122,6 +124,54 @@ set_target_properties(
   LINK_INTERFACE_LIBRARIES "" )
 
 #-------------------------------------------------------------------------------
+# The XrdSsi lib
+#-------------------------------------------------------------------------------
+add_library(
+  ${LIB_XRD_SSI}
+  SHARED
+  XrdSsi/XrdSsiDir.cc                    XrdSsi/XrdSsiDir.hh
+  XrdSsi/XrdSsiFile.cc                   XrdSsi/XrdSsiFile.hh
+  XrdSsi/XrdSsiFileReq.cc                XrdSsi/XrdSsiFileReq.hh
+  XrdSsi/XrdSsiFileSess.cc               XrdSsi/XrdSsiFileSess.hh
+  XrdSsi/XrdSsiSfs.cc                    XrdSsi/XrdSsiSfs.hh
+  XrdSsi/XrdSsiSfsConfig.cc              XrdSsi/XrdSsiSfsConfig.hh
+  XrdSsi/XrdSsiStat.cc
+)
+
+target_link_libraries(
+  ${LIB_XRD_SSI}
+  XrdSsiLib
+  XrdUtils
+  XrdServer )
+
+set_target_properties(
+  ${LIB_XRD_SSI}
+  PROPERTIES
+  INTERFACE_LINK_LIBRARIES ""
+  LINK_INTERFACE_LIBRARIES "" )
+
+#-------------------------------------------------------------------------------
+# The XrdSsiLog lib
+#-------------------------------------------------------------------------------
+add_library(
+  ${LIB_XRD_SSILOG}
+  SHARED
+  XrdSsi/XrdSsiLogging.cc
+)
+
+target_link_libraries(
+  ${LIB_XRD_SSILOG}
+  XrdSsiLib
+  XrdUtils
+  XrdServer )
+
+set_target_properties(
```

Comment:
Also, the library is installed but is not packaged.
Note this creates libXrdSsiLog.so which is installed but not packaged. libXrdSsiLog.so.* are, in turn not created but they are packaged.